application/models/UserGroup.php

```diff
@@ -231,6 +231,7 @@ public function requestViewGroup($ugId, $userId)
      * @param integer $ugId
      * @param integer $ownerId
      * @return bool
+     * @deprecated since 2018-04-21 use $this->delete and do the permissions check in controller!!
```

Comment:
Where place the 'listGroup' where dev can have : list of group with Permission usage ?